account_invoice_batches/models/common.py

```diff
@@ -7,6 +7,7 @@
 BATCH_SENDING_METHODS = [
     ("pdf", _("PDF")),
     ("email", _("e-mail")),
+    ("email_with_attachment", _("e-mail (Attach documents)")),
```

Comment:
You cannot add directly here, you should use selection_add and maybe you cannot use this common variable